src/test/init/features/functions.spec.ts

```diff
@@ -0,0 +1,236 @@
+import * as sinon from "sinon";
+import { Config } from "../../../config";
+import * as prompt from "../../../prompt";
+import { Setup } from "../../../init";
+import { doSetup } from "../../../init/features/functions";
+import { Options } from "../../../options";
+import { RC } from "../../../rc";
+import { expect } from "chai";
+import * as _ from "lodash";
```

Comment:
*nit: Unenforced but nice to follow rules when writing imports:

1) Separate out imports based on "builtin" vs "external" vs "local"

```
import foo from "fs"

import bar from "some-lib"

import car from "../local"
```

2. I tend to group named import and wildcard importt

```
import * from froo
import * from bar
import { foobar } from foobar
import { a, b, c } from alphabets
```